src/pages/home/report/ReportActionItem.tsx

```diff
@@ -454,6 +463,22 @@ function ReportActionItem({
         ];
     }, [action, report.reportID]);
 
+    const renderThreadDivider = useMemo(
+        () =>
+            !shouldHideThreadDividerLine ? (
```

Comment:
Please use `shouldHideThreadDividerLine` instead of `!shouldHideThreadDividerLine` and exchange for the corresponding conditions for more readable